xla/service/gpu/gpu_windowed_einsum_handler_test.cc

```diff
@@ -662,11 +659,11 @@ ENTRY test_main {
 
   RunAndFilecheckHloRewrite(kHloString, GpuWindowedEinsumHandler(),
                             R"(
-; CHECK-LABEL: windowed_dot_general_body_ag
```

Comment:
This test case looks autogenerated to me an I have no chance of understanding this.